ai-quick-actions/fine-tuning-tips.md

```diff
@@ -139,6 +139,109 @@ The accuracy metric reflects the proportion of correct completions made by the m
 
 As the training progresses, monitoring both accuracy and loss provides insights into the model's learning dynamics. A decreasing loss alongside increasing accuracy suggests that the model is learning effectively. However, it's important to watch for signs of over-fitting, where the model performs exceptionally well on the training data but fails to generalize to new, unseen data. This can be detected if the validation loss stops decreasing or starts increasing, even as training loss continues to decline.
 
+### Advanced Configuration Update Options
+
+The available shapes for models in AI Quick Actions are pre-configured for fine-tuning for the models available in the Fine-Tuned model tab. 
+However, if you need to add more shapes to the list of 
+available options, you can do so by updating the relevant configuration file. Currently, this 
+update option is only available for models that users can register.
+
+#### For Custom Models:
+To add shapes for custom models, follow these steps:
+
+1. **Register the model**: Ensure the model is registered via AI Quick Actions UI or CLI.
+
+2. **Navigate to the model's artifact directory**: After registration, locate the directory where the model's artifacts are stored in the object storage. 
+
+3. **Create a configuration folder**: Inside the artifact directory, create a new folder named config. For example, if the model path is `oci://<bucket>@namespace/path/to/model/`
+then create a folder `oci://<bucket>@namespace/path/to/model/config`.
+
+4. **Add a fine-tuning configuration file**: Within the config folder, create a file named `ft_config.json` with the following content:
+
+
+```
+{
+    "shape":
+    {
+        "VM.GPU.A10.1":
+        {
+            "batch_size": 1,
+            "replica": "1-10"
+        },
+        "VM.GPU.A10.2":
+        {
+            "batch_size": 1,
+            "replica": "1-10"
+        },
+        "BM.GPU.A10.4":
+        {
+            "batch_size": 1,
+            "replica": 1
+        },
+        "BM.GPU4.8":
+        {
+            "batch_size": 4,
+            "replica": 1
+        },
+        "BM.GPU.L40S-NC.4":
+        {
+            "batch_size": 4,
+            "replica": 1
+        },
+        "BM.GPU.A100-v2.8":
+        {
+            "batch_size": 6,
+            "replica": 1
+        },
+        "BM.GPU.H100.8":
+        {
+            "batch_size": 6,
+            "replica": 1
+        }
+    },
+    "finetuning_params": "--trust_remote_code True",
+    "configuration":
```

Comment:
Let us remove the configuration part as it does not play any role

Reply:
removed.